layouts/soarian/r4/clinical/diagnostics.html

```diff
@@ -0,0 +1,9 @@
+
+
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a diagnosis is confirmed, it would be recorded as a Condition so that it is part of the historical patient record.</p>
```

Comment:
In the second sentence, please change "Note," to "Note:" (Change the comma to a colon.)

PRODOC-160857

Reply:
Changed "," to colon ":"